package.json

```diff
@@ -37,7 +37,7 @@
     "clean": "lerna run clean && node packages/build/bin/run-clean \"packages/*/dist*\" \"examples/*/dist*\"",
     "clean:lerna": "lerna clean",
     "build": "node bin/run-lerna run build",
-    "build:full": "npm run clean:lerna && npm run bootstrap && npm test",
+    "build:full": "npm run clean:lerna && npm install-test",
```

Comment:
I'm seeing very strange behavior of `npm install-test`. It triggers two runs of `install` and `test`.

Reply:
See https://github.com/npm/npm/pull/20358

Reply:
Interesting. Let's hold on my pull request until npm/npm#20358 is resolved.